src/BoundingBox.cpp

```diff
@@ -36,6 +36,27 @@ bool BoundingBox::checkIntersection(BoundingBox& other) {
 		(this->min_.z <= other.max_.z && this->max_.z >= other.min_.z);
 }
 
+glm::vec3 BoundingBox::calcReflNormal(BoundingBox& other) {
+   glm::vec3 normal;
+   float epsilon = 1.5;

```

Comment:
Yeah, the epsilon needs to really big, I was surprised by that too when I did the logic originally. We might want to ask Zoe at some point whats the best for that. We should also ask her if there is a not too complicated way to make the bouncing (for basic collisions it's fine) not axis aligned